docs/messaging/gcp-pubsub.md

```diff
@@ -16,7 +16,15 @@ For Google Cloud Pub/Sub, the following additional attributes are defined:
 <!-- semconv messaging.gcp_pubsub(full,tag=tech-specific-gcp-pubsub) -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
+| `messaging.gcp_pubsub.message.ack_result` | string | The subscriber's acknowledgement of a message. If missing, the subscriber did not chose whether or not to acknowledge the message. | `ack`; `nack` | Conditionally Required: If the subscriber ACKs or NACKs the message. |
```

Comment:
For GCP at least, I think the Pub/Sub team felt the "settle" terminology would be too vague.

I think they like "ack/nack/expired" as the values for the result attribute, which I'm not sure if the other systems use.

Reply:
Also `ack_result` isn't exactly what we're looking for, we're looking for a generic `result`. The distinction is small, but setting `expired` to an attribute like `ack_result` seems to imply the `Acknowledge` call was timed out, rather than the overall message leasing timed out.